pfrl/agents/goal_conditioned_td3.py

```diff
@@ -11,6 +13,11 @@
 from pfrl.utils import clip_l2_grad_norm_
 
 
+def _mean_or_nan(xs):
+    """Return its mean a non-empty sequence, numpy.nan for a empty one."""
+    return np.mean(xs) if xs else np.nan
```

Comment:
isnt' this defined somewhere? if not, is there any other good place to put this?

Reply:
fixed.